lua/jdtls/dap.lua

```diff
@@ -291,7 +291,7 @@ local function make_config(lens, launch_args)
     vmArgs = table.concat(launch_args.vmArguments, ' ');
     noDebug = false;
   }
-  if lens.kind == TestKind.TestNG then
+  if lens.testKind == TestKind.TestNG then
```

Comment:
```suggestion
  if lens.testKind == TestKind.TestNG or lens.kind == TestKind.TestNG then
```

For compatibility with older vscode-java-test versions.